DevKoenv/poc-discord-nuxt
# Add Linters, Formatters and

> [!NOTE]
> Make sure that you have the [Bun runtime](https://bun.sh/) installed before testing as this project is made using Bun.

## Information

* Added the new `ESLint v9.8.0` to the project with the prettier and tailwind config maps. 
* Fixed all the ESLint errors that where alerted
* Fixed all the ESLint warnings that where alerted
* Refactored some classes in the CSS as they would not work in the correct tailwind order (automatic re-ordered by the tailwind config map)

## Testing

> [!WARNING]
> The ESLint scripts will emit an error, this is related to the Bun runtime and can be ignored.
> Possible Bun runtime errors due to not being implemented yet: `eval`

* Install all the packages: `bun install`
* Run the ESLint linter script: `bun run lint`, or with autofix `bun run lint:fix`
* Check if all the Typescript types are correct: `bun run type-check`
